CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Guidelines for Contribution
+
+<br>
+
+## How can I contribute in vscode-seneca-college project?
+
+<br>
+
+Never contributed to open source before? Are you curious in how contributions function in our project?
+Here is a brief summary!
+
+<br>
+
+## **Here is a brief overview of how to contribute to vscode-seneca-college:**
+
+---
+
+<br>
+
+1. Start by identifying an issue or a feature that you would want to add.
```

Comment:
Probably good to mention that contributors can file issues themselves if they have an idea that has not been documented yet or choose an existing issue from the "issues" tab.